packages/documentation-v7/.storybook/styles/preview.scss

```
@@ -148,4 +148,12 @@
       }
     }
   }
+
+  @include post.high-contrast-mode() {
+    .innerZoomElementWrapper {
+      > * {
+        border-color: Canvas;
```

```suggestion
        border-color: Canvas !important;
```

The storybook css rules has an important flag so it doesn't work as it is.

---

Additionally, anyways.

---

> The storybook css rules has an important flag so it doesn't work as it is.
> 
> Also maybe it would rather be better to make a PR in the storybook repo, no?

This is true and I thought I adressed this with an important flag as well, but maybe I just forgot to commit it.